CMakeLists.txt

```diff
@@ -78,6 +78,8 @@ if(MSVC)
         _HAS_EXCEPTIONS=0
         _UNICODE
     )
+    string(REGEX REPLACE "/[Ww][0123]" "" CMAKE_C_FLAGS "${CMAKE_C_FLAGS}")
+    string(REGEX REPLACE "/[Ww][0123]" "" CMAKE_CXX_FLAGS "${CMAKE_C_FLAGS}")
```

Comment:
I think you know cmake better than me by far. But should you be messing with the global `CMAKE_XXX` vars?

Reply:
See https://cmake.org/cmake/help/latest/variable/CMAKE_LANG_FLAGS.html
This variable is initialized by the environment variable `CFLAGS`,
so what I'm doing is overriding the default/user flags.

Reply:
Also note that this override is done in the crashpad directory.
So it will not be applied to sentry-native or user projects.

Reply:
You're correct that modifying variables such as `CMAKE_C_FLAGS` is frowned upon, as targets are preferred. 
There are functions to add options, but by my knowledge no functions to remove/filter/initialize options.

Reply:
> Also note that this override is done in the crashpad directory.
> So it will not be applied to sentry-native or user projects.

If that is the case, then I think its fine. Lets just say I had a bad experience with flags and all kinds of other things leaking out where they shouldn’t 🤣